src/deluge/gui/views/clip_view.cpp

```diff
@@ -29,16 +29,23 @@
 #include "model/clip/clip_minder.h"
 #include "gui/views/view.h"
 #include "definitions.h"
+#include "model/consequence/consequence_clip_horizontal_shift.h"
+#include "memory/general_memory_allocator.h"
+#include <new>
+
+static Clip* getCurrentClip() {
+  return currentSong->currentClip;
+}
```

Comment:
If it's not too much trouble, having this refactor pulled out to a separate kit would be nice (then when tracing history we can differentiate between the feature work and the refactor).

Reply:
Just a separate commit, same PR?

Reply:
Sounds great, thanks!